src/Html/Tags.elm

```diff
@@ -2,8 +2,9 @@ module Html.Tags (..) where
 
 import Json.Encode exposing (string)
 import VirtualDom exposing (Node, property)
-import Html.Attributes exposing (attribute)
-import Html exposing (div, Html)
+import Html.Attributes exposing (attribute, href, action, src)
+import Html exposing (div, Html, Attribute)
+import Shared.Routes exposing (Route, routePath, Style, File, stylePath, filePath)
```

Comment:
I don't think this file should be importing anything from routes. You can make the various helpers live elsewhere (or make a function that returns them for a given project)